sdk/nodejs/tests/runtime/closure.spec.ts

```diff
@@ -5128,6 +5128,70 @@ return function /*f*/({ whatever }) { };
 `,
     });
 
+    {
+        const regex = /(abc)[\(123-456]\\a\b\z/gi;
+
+        cases.push({
+            title: "Regex #1",
+            // @ts-ignore
+            func: function() { console.log(regex); },
+            expectText: `exports.handler = __f0;
+
+var __regex = new RegExp("(abc)[\\\\(123-456]\\\\\\\\a\\\\b\\\\z", "gi");
```

Comment:
this output is what we want. First, the backslashes are doubled because we're in a JS string. So consider the number of backslashes halved. Now, this is also code that contains a JS string with the ecsaped regex contents. So that string should have double the backslashes of the original regex.